src/components/data/SelectionDrawer.js

```diff
@@ -86,12 +87,21 @@ function SelectionToolbar(props) {
         onConfirm(currentPath);
     };
 
-    const invalidTotal =
-        ResourceTypes.ANY !== acceptedType
-            ? selectedResources.filter(
-                  (resource) => resource.type.toLowerCase() !== acceptedType
-              ).length
-            : 0;
+    let invalidTotal = 0;
+    if (ResourceTypes.FOLDER === acceptedType && !multiSelect) {
+        if (
+            selectedResources[0]?.infoType !== InfoTypes.HT_ANALYSIS_PATH_LIST
+        ) {
+            invalidTotal = 1;
+        }
+    } else {
+        invalidTotal =
+            ResourceTypes.ANY !== acceptedType
+                ? selectedResources.filter(
+                      (resource) => resource.type.toLowerCase() !== acceptedType
+                  ).length
+                : 0;
+    }
```

Comment:
I think this could be simplified a bit, if we store some of these boolean checks into variables.
For example
```suggestion
    const allowAny = ResourceTypes.ANY !== acceptedType;
    const allowPathList = ResourceTypes.FOLDER === acceptedType && !multiSelect;
    let invalidTotal = 0;
    if (!allowAny) {
        invalidTotal = selectedResources.filter((resource) =>
                       resource.type.toLowerCase() !== acceptedType
                       && !(allowPathList && resource.infoType === InfoTypes.HT_ANALYSIS_PATH_LIST)
                  ).length;
    }
```

Reply:
thanks the suggestion although I am more inclined to use code I wrote since I spent some time coming up with the logic and testing them.